guides/getting-started.md

```diff
@@ -1,196 +1,73 @@
 
-Once the animations package is installed, import `BrowserAnimationsModule` into your application to enable animations support.
+1. Choose a prebuilt theme name, or "custom" for a custom theme ? 
+You can choose from [prebuilt material design themes](https://material.angular.io/guide/theming#using-a-pre-built-theme) or define an extensible custom theme. 
 
-```ts
-import {BrowserAnimationsModule} from '@angular/platform-browser/animations';
+2. Set up HammerJS for gesture recognition ?
+HammerJS provides [gesture recognition capabilities](https://material.angular.io/guide/getting-started#step-5-gesture-support) you can use in your apps. Angular Material requires HammerJS only for the mat-slider component.
 
-@NgModule({
-  ...
-  imports: [BrowserAnimationsModule],
-  ...
-})
-export class PizzaPartyAppModule { }
-```
+3. Set up browser animations for Angular Material?
+Importing the BrowserAnimationsModule into your application enables Angular's animation system. Omitting this will disable most of Angular Material's animations.
 
-Alternatively, you can disable animations by importing `NoopAnimationsModule`.
+The `ng add` command will additionally perform the following configurations:
 
-```ts
-import {NoopAnimationsModule} from '@angular/platform-browser/animations';
+* Adds dependencies to package.json
+* Add the Robot font to your index.html
+* Add the Material Design icon font to your index.html
+* Adds a few minimal global CSS resets to:
+  * Remove margins from body
+  * Set height: 100% on html and body
+  * Set Roboto as default application font
 
-@NgModule({
-  ...
-  imports: [NoopAnimationsModule],
-  ...
-})
-export class PizzaPartyAppModule { }
-```
+You're done! Angular Material is now configured to be used in your application.
 
-### Step 3: Import the component modules
+### Step 2: Display an example Angular Material component
 
-Import the NgModule for each component you want to use:
+Let's display a  slider component in your app and verify that everything works.
 
-```ts
-import {MatButtonModule} from '@angular/material/button';
-import {MatCheckboxModule} from '@angular/material/checkbox';
-
-@NgModule({
-  ...
-  imports: [MatButtonModule, MatCheckboxModule],
-  ...
-})
-export class PizzaPartyAppModule { }
-```
-
-Alternatively, you can create a separate NgModule that imports and then re-exports all of the Angular Material components that you will use in your application. By exporting them again, other modules can simply include your `CustomMaterialModule` wherever Material components are needed, and automatically get all of the exported Material modules. A good place for importing/exporting the application-wide Material modules is the [SharedModule](https://angular.io/guide/ngmodule-faq#sharedmodule).
+You need to import the `MatSliderModule` that you want to display by adding the following lines to your app.module.ts file.
```

Comment:
I think that it may be a better getting started experience for new users if they generated a nav via the nav schematic (`ng g @angular/material:nav nav`).

They would only need one additional step to replace their `app.component.html` with the `<app-nav></app-nav>` component after they created it.

Reply:
I discussed this with Jeremy and we decided to go with the MatSlider as the nav looks a bit out of place on the demo page. I'm back in the office and will address the other comments and submit my changes.